app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :events, foreign_key: 'creator_id', class_name: 'Event'
```

Comment:
`class_name` is not needed here. class should be configured on its own given the name used is `:events` had it been different, then yes